dandi/dandiapi.py

```diff
@@ -318,10 +318,16 @@ def __init__(
         self, api_url: Optional[str] = None, token: Optional[str] = None
     ) -> None:
         """
-        Construct a client instance from a server's base API URL and an
-        optional authentication token/API key.  If no URL is supplied, the URL
-        of the instance named in the :envvar:`DANDI_INSTANCE` environment
-        variable (default value: ``dandi``) is used.
+        Construct a client instance.
+
+        :param api_url: API URL. For DANDI production, use ``"dandi"``. For DANDI staging,
+        use ``"https://api-staging.dandiarchive.org/api"``. If no URL is supplied, the
+        value of the  :envvar:`DANDI_INSTANCE` environment variable is used. If that
```

Comment:
"the value of the :envvar:`DANDI_INSTANCE` environment variable is used." — The value is not used directly; rather, the API URL of the instance whose name is stored in the envvar is used.